src/app/api/api.py

```diff
@@ -333,6 +364,15 @@ async def filter_pending_collections(
         },
     ),
 ):
+    """
+    A list of missing entries for different types of materials by sub-collection.
```

Comment:
On line 336 it is called child collection, here sub-collection. I am fine with both namings, but I suggest we decide on one. I prefer the child collection, since it goes in line with "children".

Reply:
Changed most of the childs to subs (as it is also used in the endpoint `/collections/{node_id}/pending-subcollections/{missing_attribute}`.

I would argue there is a minimal semantic difference, but i don't we are at the level where these subtile things matter :D
 - Child is a "direct subnode"
 - subnode could also be a grandchild.